### DISCUSSION TITLE
Ton of BTT boards on sale at amazon

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#41] [Show and tell]
https://www.amazon.com/BIGTREETECH-Octopus-8xTMC2209-Controller-Raspberry/dp/B09RWRBH47/ref=mp_s_a_1_7

Just a heads up. I think I will snag another octopus

**Comment by jon-harper** [comment]
OoOoh, [even the Manta is on sale](https://www.amazon.com/BIGTREETECH-Integrated-Motherboard-Firmware-Compatible/dp/B0B7W53JSY/). That's brand new.
